src/fe-common/irc/fe-irc-messages.c

```diff
@@ -37,28 +37,47 @@
 #include "fe-queries.h"
 #include "window-items.h"
 
-static const char *skip_target(IRC_SERVER_REC *server, const char *target)
+int fe_channel_is_opchannel(IRC_SERVER_REC *server, const char *target)
 {
-	int i = 0;
-	const char *val, *chars;
+	const char *statusmsg;
+
+	/* Quick check */
+	if (server == NULL || server->prefix[(int)(unsigned char)*target] == 0)
+		return FALSE;
+
+	statusmsg = g_hash_table_lookup(server->isupport, "statusmsg");
```

Comment:
Even though this is case insensitive, i'm not really sold on having it as lowercase, given that it's looking up an ISUPPORT value that is uppercase. But yeah. Nitpick.